tests/wicked/basic/sut/t05_dynamic_addresses_xml.pm

```diff
@@ -22,6 +22,7 @@ use utils 'file_content_replace';
 sub run {
     my ($self, $ctx) = @_;
     my $config = '/data/dynamic_address/dynamic-addresses.xml';
+    sleep 30;    # W/A give ovs some time for setup !?
```

Comment:
I don't like this! 
My guess is, that the ovs-switch isn't fully aware that the VM restarted. And drop somehow packages.